artifact_requirements.txt

```diff
@@ -11,7 +11,11 @@ pandas
 pyyaml
 scipy
 tables
-vivarium_inputs[data]>=5.0.0
+vivarium_inputs>=5.0.0
+# Note inputs is not install with [data] extra because we might need to specify the branch
+# for inputs for the data requirements
+vivarium_gbd_access>=4.0.0
+core-maths
```

Comment:
The comments in the requirements files aren't meant to be vague but we can't include of of the strings we grep for otherwise it will show up in the grep outputs we iterate through.